data/docker_packages.txt

```diff
@@ -1,50 +1,124 @@
-python
-php
-tomcat
-wordpress
-haproxy
-crate
-java
-nextcloud
-telegraf
-cassandra
-logstash
-gradle
-ghost
-kong
-solr
-jenkins
-vault
-swarm
-rocket.chat
-joomla
-adminer
-redmine
-mono
-pypy
-elixir
-composer
-matomo
-gogs/gogs
-pihole/pihole
-minio/minio
-gitlab/gitlab-runner
-bitnami/mongodb
-v2tec/watchtower
-google/cadvisor
-grafana/grafana
-prom/prometheus
-cockroachdb/cockroach
-atmoz/sftp
-hashicorp/terraform
-gitlab/gitlab-ce
\ No newline at end of file
+couchbase:latest
+redis:latest
+redis:stretch
+mongo:3.4-xenial
+nginx:1.15
+ubuntu:18.04
+ubuntu:16.04
+busybox:glibc
+busybox:uclibc
+node:8.16.0-jessie
+node:lts
+node:11
+mysql:latest
+mysql:5.6
+httpd
+httpd:2.4
+hello-world
+postgres:latest
+postgres:10
+postgres:9
+alpine:latest
+alpine:3.7
+memcached:1.5.14-alpine
+traefik:2.0
+traefik:v1.7.11-alpine
+mariadb:5.5
+mariadb:10.2.24-bionic
+mariadb:rc
+consul:1.4
+centos:6
+centos:latest
+golang:stretch
+php:7.4-fpm
+php:7.3.5-zts-alpine3.8
+php:7.1.29-zts-jessie
+elasticsearch:6.7.2
+influxdb:1.6
+python:3-stretch
+python:3.6.8-windowsservercore-ltsc2016
+haproxy:1.6.14-alpine
+wordpress:php7.1-fpm-alpine
+nextcloud:14-fpm
+ghost:latest
+kong:1.1.2-centos
+amazonlinux:2018.03.0.20190212
+eclipse-mosquitto:1.5
+jenkins:latest
+gradle:5.4.1-jre8-alpine
+gradle:5.4-jre
+bash:4.0.44
+adminer:4.7.1-fastcgi
+redmine:4-passenger
+matomo:fpm-alpine
+swift:5.0-bionic
+composer:1.8.5
+rust:latest
+rust:1.34.2-stretch
+rust:1-slim
+mediawiki:latest
+yourls:1.7.3-fpm
+cfcommunity/slack-notification-resource:latest
+v2tec/watchtower:armhf-0
+portainer/portainer:windows-amd64
+google/cadvisor:v0.31.0
+kope/protokube:1.4.2
+gliderlabs/registrator:edge
+kylemanna/openvpn:2.3
+grafana/grafana:6.1.4
+linuxserver/radarr:arm32v7-latest
+prom/prometheus:v2.9.2
+minio/minio:7664-5812d7f
+gitlab/gitlab-runner:latest
+gliderlabs/logspout:latest
+msvechla/vaultbot:v1.8
+cockroachdb/cockroach:v19.1.1
+linuxserver/ombi:amd64-latest
+atmoz/sftp:debian-jessie
+linuxserver/tautulli:v2.1.29-ls17
+bitnami/redis:5.0.5
+mesosphere/aws-cli:latest
+hashicorp/terraform:0.11.14
+linuxkit/kernel:4.9.175-amd64
+linuxserver/plex:arm64v8-latest
+pdffiller/registrator:latest
+centurylink/watchtower:latest
+bobrik/curator:5.7.6
+mobingi/alm-awslogs:latest
+buoyantio/linkerd:latest-openj9-experimental 
+rancher/agent:v1.2.11
+istio/mixer:1.1.7
+weaveworksdemos/orders:latest
+linuxserver/heimdall:arm64v8-latest
+homeassistant/home-assistant:0.93.1
+drone/drone:linux-amd64
+circleci/ruby:2.5-node-browsers-legacy
+phpmyadmin/phpmyadmin:latest
+pihole/pihole:4.3
+gogs/gogs:latest
+sguha/scripthost:0.1.4.4148
+rancher/server:stable
+envoyproxy/envoy-alpine:latest
+portainer/agent:latest
+linuxserver/letsencrypt:latest
+linuxserver/letsencrypt:0.34.2-ls29
+avinetworks/se:latest
+emby/embyserver:4.2.0.6
+lsiocommunity/organizr:132
+jacobalberty/unifi:5.10.23
+skynetservices/skydns:2.5.2b
+pebbletech/cloudwatch-stats:python
+anchorfree/afmon:v1.3.0
+wurstmeister/kafka:2.12-2.2.0
```

Comment:
Wouldn't it be fun to keep the tags and the names separate and then generate them on top of each other? You'd get nonsensical stuff like `_/ubuntu:2.5-node-browsers-legacy` and that's kinda unrealistic but also fun. What do you think?